skribenten-backend/src/main/kotlin/no/nav/pensjon/brev/skribenten/routes/Brevmal.kt

```diff
@@ -35,12 +35,14 @@ fun Route.brevmal(brevbakerService: BrevbakerService, brevmalService: BrevmalSer
 
             get("/modelSpecification") {
                 val brevkode = call.parameters.getBrevkode()
-                brevbakerService.getModelSpecification(brevkode)
-                    .onOk { call.respond(it) }
-                    .onError { message, status ->
-                        logger.error("Feil ved henting av modelSpecification for ${brevkode.kode()}: Status:$status Melding: $message ")
-                        call.respond(status, message)
-                    }
+                val modelSpec = brevbakerService.getModelSpecification(brevkode)
```

Comment:
Reknar med det er med vilje du let ein eventuell exception propagere vidare no?

Trur det gir meining, vi forventar jo ikkje at det skal skje, og har ikkje heilt fornuftig handtering på plass for det.

Reply:
Ja, den vil bli plukket opp av den generelle `Exception` status-page i appen.